tap_github.py

```diff
@@ -9,15 +9,16 @@
 logger = singer.get_logger()
 
 
-def authed_get(source, url):
+def authed_get(source, url, headers={}):
     with singer.stats.Timer(source=source) as stats:
+        session.headers.update(headers)
```

Comment:
these headers will be stuck in the session the next time it's used, right?  That's probably going to burn the next person who touches this tap.  Can you instead add the headers to the `session.request` call below?